Detector/DetSegmentation/src/GridPhiEtaHandle.cpp

```diff
@@ -0,0 +1,86 @@
+// Framework include files
+#include "DetSegmentation/GridPhiEtaHandle.h"
+#include "DD4hep/objects/SegmentationsInterna.h"
+#include "DetSegmentation/GridPhiEta.h"
+#include "DD4hep/Handle.inl"
+
+using namespace std;
+using namespace DD4hep::Geometry;
+
+/// determine the position based on the cell ID
```

Comment:
repeating docs appearing in the .h files here is nice, but has the risk of running out of sync.